include/libopencm3/pac55xx/ccs.h

```diff
@@ -26,18 +27,100 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with this library.  If not, see <http://www.gnu.org/licenses/>.
  */
-#ifndef INCLUDE_LIBOPENCM3_PAC55XX_CCS_H_
-#define INCLUDE_LIBOPENCM3_PAC55XX_CCS_H_
+#ifndef LIBOPENCM3_PAC55XX_CCS_H_
+#define LIBOPENCM3_PAC55XX_CCS_H_
 
 #include <libopencm3/cm3/common.h>
+#include <libopencm3/cm3/memorymap.h>
+#include <libopencm3/pac55xx/memorymap.h>
+#include <libopencm3/pac55xx/memctl.h>
 
 /**@{*/
 
-/** Clock Control Registers
- * @defgroup clock_config_regs Clock Config Registers.
- * @{*/
+/** @defgroup ccs_frequencies CCS Frequencies
+@{*/
+/** Ring Oscillator Frequency */
+#define CCS_ROSC_FREQ       (16000000ul)
```

Comment:
opencm "standard" would seem to be caps literal type definitions, so should be `16000000U`  (`long` is redundant as this will _always_ be a 32-bit machine, so would drop to match with more "standard" formatting)

Reply:
changed suffix to cap U